File: EQIS_env-main/react_sample/springboot/Spring/src/main/java/com/sample/basic/sample/code/model/MlgMsgB.java
package com.sample.basic.sample.code.model;

import com.sample.basic.cmm.model.Base;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

/**
 * <pre>
 * 
 * </pre>
 * 
 * @author user
 *
 */

@Data
@EqualsAndHashCode(callSuper=false)
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class MlgMsgB extends Base {
	
	private String mlgCd;
	private String langCd;
	private String mlgSbc;
	private String useYn;

}

File: EQIS_env-main/react_sample/react/src/interfaces/lang/mlgMsgB.ts
import { cloneDeep } from 'lodash';

import { baseInit, Ibase } from '../common/base';

export interface ImlgMsgB extends Ibase {
    mlgCd: string;
    langCd: string;
    mlgSbc: string;
    useYn: string;
}

export const mlgMsgBInit = () => {
    const mlgMsgB: ImlgMsgB = {
        mlgCd: '',
        langCd: '',
        mlgSbc: '',
        useYn: '',
        ...baseInit(),
    };

    return cloneDeep(mlgMsgB);
};
